patterns/2-structured/circle-communities.md

```diff
@@ -0,0 +1,81 @@
+
+* **Define the Purpose**: Clearly articulate the purpose and goals of the Circle Community.  Ensure that these align with the broader mission of your organization or initiative.
+* **Establish Social Conventions**: Develop and communicate social conventions that promote respect, inclusivity and open dialogue.  Encourage members to embrace and evolve these guidelines to create a safe and supportive environment.
+* **Facilitate Regular Meetings**: Schedule regular meetings (e.g., weekly) to maintain engagement and momentum.  Use these meetings to discuss progress, share insights and address challenges.  Meetings are meant to be open for people to put things "onto the table" for unpacking.  If there are specific goals and objectives to complete, then they must be negotatied by the attending group.
+* **Rotate Leadership**: Encourage shared leadership by rotating the facilitation role among members.  This helps distribute responsibility and fosters a sense of connection.
+* **Use Collaborative Tools**: Leverage collaborative tools and platforms (e.g., Jitsi Meet, Slack, GitHub) to facilitate communication, documentation and knowledge sharing.  Insights learned can be optionally shared to social channels such as Slack and LinkedIn, ensuring privacy.
+* **Foster a Culture of Continuous Improvement**: Regularly solicit feedback from members to identify areas for improvement. Use this feedback to refine the conditioning and activities of the Circle Community.
+
+## Resulting Context
+
+Embracing this pattern helps InnerSource programs become vibrant, effective and sustainable. Participants will feel more connected and engaged, leading to increased knowledge sharing, innovation and collective growth.
+
+## Related Patterns
+
+* [Dedicated Community Leader](https://patterns.innersourcecommons.org/p/dedicated-community-leader)
+* [Core Team](https://patterns.innersourcecommons.org/p/core-team)
+* [Maturity Model](https://patterns.innersourcecommons.org/p/maturity-model)
+* [Agile InnerSource Dojo](https://patterns.innersourcecommons.org/p/agile-innersource-dojo)
+
+## Known Instances
+
+* Dojo Circle
+  * Fridays at 13:00 UTC at <https://circle.dojo.center>
+  * Hosted by [Dojo Center](https://dojo.center)
+  * Engaged via [InnerSource Commons Foundation](https://innersourcecommons.org) in Slack at ```#dojo-center```
+  * Affiliated with the Mindset, Agile and InnerSource sub-communities within _InnerSource Dojo from SAP_
+* [InnerSource Dojo from SAP](https://www.youtube.com/watch?v=fXoVm5iTSCc)
+* _Unplugged InnerSource_ sessions within the _Reuse & InnerSource_ workgroup of [SAP Cross-Platform Architecture](https://community.sap.com/t5/technology-blogs-by-sap/cross-product-architecture-embracing-conway-s-law-for-better-software/ba-p/13648600)
```

Comment:
```suggestion
* _Unplugged InnerSource_ sessions within the _Reuse & InnerSource_ workgroup of [SAP Cross-Platform Architecture](https://community.sap.com/t5/technology-blogs-by-sap/cross-product-architecture-embracing-conway-s-law-for-better-software/ba-p/13648600)
* [Guilds at the BBC](https://youtu.be/MyYgk-jdjSw?t=511)
```

We've done deeper work on communities - I'll see if I can find a better reference - but this might be an OK known instance anyway?